packages/swarm-docs/src/components/propstable.jsx

```diff
@@ -21,25 +21,32 @@ const PropsTable = props => {
 
           return (
             <tr key={key}>
-              <td style={{ color: 'rgb(17, 147, 154)' }}>{key}</td>
+              <td><span className="text-code">{key}</span></td>
               <td>{prop.flowType.name}</td>
               {prop.required ? (
-                <td style={{ color: 'rgb(255, 76, 34)' }}>required</td>
+                <td><span className="text-required">required</span></td>
```

Comment:
trivial, but I recommend putting the `<td>` outside the ternary and just rendering the `span` inside the ternary. Alternatively, can you just pass the `text-required` className to the `<td>` and avoid the span entirely? It looks intentional but I'm not sure why it would affect layout